src/test/java/com/SAS/facility/FacilityIT.java

```diff
@@ -0,0 +1,73 @@
+package com.SAS.facility;
+
+import com.SAS.game.Game;
+import com.SAS.team.Team;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FacilityIT {
+
+    private Facility facility;
+    private Game game;
+    private Team team;
+
+    @BeforeEach
+    void setUp() {
+        facility = new Facility();
+        game = new Game();
+        team = new Team();
+    }
+
+    @Test
+    void addValidGameToFacilityBooleanCheck() {
+        facility.addGame(game);
+        boolean result = facility.addGame(game);
+        assertTrue(result);
+    }
+
+    @Test
+    void addValidGameToFacilityListCheck() {
+        facility.addGame(game);
+        int result = facility.getGamesList().size();
+        assertEquals(1, result);
```

Comment:
Maybe you should clear the games list first, since it will not necessarily have a size of 1 if you added more games in other tests?
You can also just check that the returned list contains the game

Reply:
The facility object gets initialize every test so its only effected by this test.